interface/main/tabs/js/tabs_view_model.js

```diff
@@ -372,6 +373,29 @@ function menuActionClick(data,evt)
 
 }
 
+function reportMenuClickData(data, evt = 'menuClick') {
+    top.restoreSession();
+    const clickEventData = {
```

Comment:
I think this looks pretty good.  We can build on this to grab the AuditEvents where type is equal to 'http-request' and post those to the tracker at a later date as well.